testsuite/features/support/commonlib.rb

```diff
@@ -417,15 +417,20 @@ def file_inject(node, local_file, remote_file)
   node.inject(local_file, remote_file, 'root', false)
 end
 
-# This function updates the server certificate on the controller node
+# This function updates the server certificate on the controller node by
+# - deleting the old one from the nss database
+# - importing the new one from the server
 def update_controller_ca
   server_ip = get_target('server').public_ip
   server_name = get_target('server').full_hostname
 
-  puts `rm /etc/pki/trust/anchors/*;
+  command = `certutil -d sql:/root/.pki/nssdb -t TC -n "susemanager" -D;
+  rm /etc/pki/trust/anchors/*;
   wget http://#{server_ip}/pub/RHN-ORG-TRUSTED-SSL-CERT -O /etc/pki/trust/anchors/#{server_name}.cert &&
   update-ca-certificates &&
   certutil -d sql:/root/.pki/nssdb -A -t TC -n "susemanager" -i  /etc/pki/trust/anchors/#{server_name}.cert`
+
+  puts(command)
```

Comment:
debug output to remove ?

or use "log" ?

Reply:
Thank you. I thought it would be a good solution to have the output, but I removed it in the end.